src/codeu/chat/ServerMain.java

```diff
@@ -34,6 +43,8 @@
 final class ServerMain {
 
   private static final Logger.Log LOG = Logger.newLog(ServerMain.class);
+  public static File persistentPath = null;
+  public Queue<String> persistentLog = new LinkedList<String>();
```

Comment:
Is it generally good practice to have just a Queue and instantiate a LinkedList or have a LinkedList instantiate a LinkedList? @wrdana @mattiewasiak